packages/contracts-bedrock/src/dispute/AnchorStateRegistry.sol

```diff
@@ -23,105 +22,222 @@ import { ISuperchainConfig } from "interfaces/L1/ISuperchainConfig.sol";
 ///         challenged within the challenge period. By using stored anchor states, new FaultDisputeGame instances can
 ///         be initialized with a more recent starting state which reduces the amount of required offchain computation.
 contract AnchorStateRegistry is Initializable, ISemver {
-    /// @notice Describes an initial anchor state for a game type.
-    struct StartingAnchorRoot {
-        GameType gameType;
-        OutputRoot outputRoot;
-    }
+    error InvalidGame();
+    error AnchorGameIsNewer(uint256 anchorGameL2BlockNumber, uint256 candidateGameL2BlockNumber);
+
+    event AnchorGameSet(IDisputeGame newAnchorGame);
+    event RespectedGameTypeSet(GameType gameType);
+    event DisputeGameBlacklisted(IDisputeGame game);
+    event GameRetirementTimestampSet(uint64 timestamp);
 
     /// @notice Semantic version.
+    // TODO: update?
```

Comment:
when do we update `version`?

Reply:
I'd consider this a major version bump. So we'd bump to 3.0.0-beta.1.